### ISSUE TITLE
Use the same address twice: error_message -> Must specify a given destination address only once.

### ISSUE THREAD

**Issue by DmitriyKapeliukh** [BlockIo/block_io-java#4]
Case: Destination addresses is the same address defined twice.

POST: prepare_transaction/?api_key={key}&amounts=0.002,0.002&to_addresses={ADDRESS1,ADDRESS1}

I've got an error from API: 
```
"data": {
        "error_message": "Must specify a given destination address only once."
    }
```

Is this case valid for BlockIO API?

**Comment by nwasiq** [comment]
I'm not sure I understand. You're entering the same destination address twice, is that intentional? If yes, why?

**Comment by DmitriyKapeliukh** [comment]
Yes, entering the same address twice. I want to test some cases in my crypto wallet. Why does your API throw this exception, this is not a valid case for BTC network?
